tests/modules/nf-core/falco/test.yml

```diff
@@ -0,0 +1,29 @@
+- name: falco test_fastqc_single_end
+  command: nextflow run ./tests/modules/nf-core/falco -entry test_fastqc_single_end -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/falco/nextflow.config
+  tags:
+    - falco
+  files:
+    - path: output/falco/fastqc_data.txt
+      md5sum: c5d0cd529d1d2f5a842bf3afd06a9567
+    - path: output/falco/fastqc_report.html
+      contains: "[ # TODO nf-core: file md5sum was variable, please replace this text with a string found in the file instead ]"
```

Comment:
These need fixing (and below)

Reply:
One can either remove this and keep the `md5sum` if the `md5sum` does not differ every time running the module